CHANGELOG.md

```diff
@@ -1,9 +1,21 @@
+3.1.0 (June 2017)
+-----------------
+
+- Add support for markdown in panel labels (PR #209)
```

Comment:
"Add support for italics and bold in panel labels using markdown syntax" might be nicer for users not familiar with markdown.